lib/talkable/generators/install_generator.rb

```diff
@@ -1,15 +1,12 @@
+require 'talkable/generators/base_generator'
+
 module Talkable
-  class InstallGenerator < Rails::Generators::Base
+  class InstallGenerator < BaseGenerator
     source_root File.expand_path("../templates", __FILE__)
-    class_option :haml, type: :boolean, default: false
-    class_option :slim, type: :boolean, default: false
 
     def ask_config_values
       @site_slug  = ask("Your Talkable site slug:")
       @api_key    = ask("Your Talkable API Key:")
-      if yes?('Do you have a custom domain? [Y/n]')
-        @server   = ask("Your custom domain [#{Talkable::Configuration::DEFAULT_SERVER}]:")
```

Comment:
almost all enterprise customer require to have a white labeled domain in their integration libraries and also this config is very helpful for development environment

Reply:
didn't know about the white labeled domain part, my perception was that everyone who uses talkable (except our dev team) is using talkable.com.
I'm rolling back these all configuration changes.